lib/search/advanced-search.ts

```
@@ -222,21 +226,27 @@ export class AdvancedSearchEngine extends UnifiedSearchEngine {
+          }
           break
         case "company_name":
-          compareValue = a.company.name.localeCompare(b.company.name)
+          if (a.company && b.company) {
+            compareValue = a.company.name.localeCompare(b.company.name)
+          }
           break
         case "amount":
           const amountA =
-            a.summaryOfBenefits?.reduce((sum, sob) => sum + (sob.amountToDraft || 0), 0) || 0
+            a.summaryOfBenefits?.reduce((sum, sob) => sum + (sob.amount_to_draft || 0), 0) || 0
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)